packages/adapter-planetscale/src/conversion.ts

```diff
@@ -101,5 +101,12 @@ export const cast: typeof defaultCast = (field, value) => {
   if (field.type === 'JSON' && value === 'null') {
     return JsonNullMarker
   }
-  return defaultCast(field, value)
+
+  const defaultValue = defaultCast(field, value)
+
+  if (defaultValue instanceof Uint8Array) {
+    return Array.from(defaultValue)
+  }
+
+  return defaultValue
```

Comment:
This will probably end up in PlanetScale's upstream, but for the time being it lets the TypeScript functional Client test `driver-adapters/team-orm-687-bytes` pass.